packages/world/src/Utils.sol

```diff
@@ -20,10 +20,12 @@ import { TerrainLib } from "./systems/libraries/TerrainLib.sol";
 
 import { EntityId } from "./EntityId.sol";
 
+// TODO: replace with VoxelCoordLib.toVoxelCoord
```

Comment:
Lets resolve these in this PR since you've already added the function to VoxelCoordLib